bip-schnorr.mediawiki

```diff
@@ -149,22 +149,22 @@ All provided signatures are valid with overwhelming probability if and only if t
 ==== Public Key Generation ====
 
 Input:
-* The secret key ''d'': an integer in the range ''1..n-1'' chosen uniformly at random.
+* The secret key ''sk'': an integer in the range ''1..n-1'' chosen uniformly at random.
 
-The public key corresponding to secret key ''d'' is ''bytes(dG)''.
+The public key corresponding to secret key ''sk'' is ''bytes(skG)''.
```

Comment:
In commit "secret key is sk not d'":

I'm not sure `skG` is more readable; it's easy to confuse it with `s*kG` (which is moderately sensible even).

Reply:
Yeah. I don't think `sk*G` is a great change either. But `bytes(int(sk)G)` isn't too bad, if converting sk to a byte array rather than an int directly works?

Reply:
Not really convinced by using multi character names. We use `ed` to mean multiplication. Could use `x` if you don't like `d`? ACK switching from accepting secret keys as bytes instead of ints.